common/src/main/scala/org/apache/comet/vector/NativeUtil.scala

```diff
@@ -96,6 +96,30 @@ class NativeUtil {
 
     (0 until batch.numCols()).foreach { index =>
       batch.column(index) match {
+        case selectionVector: CometSelectionVector =>
```

Comment:
The code for `CometSelectionVector` and `CometVector` is almost identical. A helper method `exportSingleVector` was added that is also similar. Was the intention to call that rather than duplicate this code?